.circleci/config.yml

```diff
@@ -35,7 +35,7 @@ aliases:
       - image: circleci/openjdk:8-jdk-node
         environment:
           # https://circleci.com/docs/2.0/java-oom/
-          _JAVA_OPTIONS: "-Xms128m -Xmx2g"
+          _JAVA_OPTIONS: "-Xms128m -Xmx1536m"
```

Comment:
In ScalaJS tests not all the memory is controlled by JVM. The idea is to allocate less to give some memory to the underlying processes. The same setting is used in GeoTrellis GDAL tests.